### PR TITLE
set admin user from a rake task and disable first-user is admin

### LINKED ISSUE #313: set admin user from a rake task and disable first-user is admin
For security reason, I would propose that we have an option to disable the "first-user is admin" feature and instead add a rake task to set the admin user.

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
fixes #313

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
I've set false as default because it is the safest option, but it breaks backward compatibility ... is it ok?

File: config/config.yml [hunk 1]
```diff
@@ -39,3 +39,14 @@ ldap:
   guess_email:
     enabled: false
     attr: ""
+
+# Set first_user_admin to true if you want that the first user that signs up
+# becomes the admin.
```
Comment (flavio) [inline config/config.yml]: `Set first_user_admin to true if you want that the first user that signs up to be an admin.`

**Comment by flavio** [comment]
yeah, I'm a bit sceptical about breaking the compatibility. I wouldn't do it

**Comment by mssola** [comment]
Same here, default to true. Moreover, the config file is wrong, it should have something set always, so comment out the code. It should be like this:

```yaml
# Set to true if you want that the first user that signs up to be an admin. If
# this is false, then you will need to perform the following command to set the admin user:
#  $ rake portus:set_admin
first_user_admin:
  enabled: true
```

Then, when checking about this in the Ruby code, yo do:

```ruby
APP_CONFIG["first_user_admin"].enabled?
```

**Comment by mssola** [comment]
Oh, and by the way, could you update the Changelog ?

**Comment by jordimassaguerpla** [comment]
does it look good now :)

File: config/config.yml [hunk 2]
```diff
@@ -39,3 +39,14 @@ ldap:
...
     attr: ""
+
+# Set first_user_admin to true if you want that the first user that signs up
+# to be an admin.
+#
+# Set to false otherwise. Then you will need to run
+#   rake portus:set_admin
+# in order to set the admin user
+#
+
```
Comment (mssola) [inline config/config.yml]: empty line not needed

File: config/config.yml [hunk 3]
```diff
@@ -39,3 +39,14 @@ ldap:
...
     enabled: false
     attr: ""
+
+# Set first_user_admin to true if you want that the first user that signs up
+# to be an admin.
+#
+# Set to false otherwise. Then you will need to run
+#   rake portus:set_admin
+# in order to set the admin user
+#
```
Comment (mssola) [inline config/config.yml]: not needed

**Comment by mssola** [comment]
LGTM

**Comment by flavio** [comment]
+1

**Pull request merged**